crates/cheatnet/tests/cheatcodes/roll.rs

```diff
@@ -144,9 +150,11 @@ fn roll_stop() {
 
     let old_block_number = recover_data(output);
 
-    runtime_state
-        .cheatnet_state
-        .start_roll(CheatTarget::One(contract_address), Felt252::from(123_u128));
+    runtime_state.cheatnet_state.start_roll(
+        CheatTarget::One(contract_address),
+        Felt252::from(123_u128),
+        CheatSpan::Indefinite,
```

Comment:
I think it'd be easier to actually leave start_roll as-is and have another method for a roll with a span